requirements-mac.txt

```diff
@@ -1,5 +1,5 @@
 wxpython
-pywin32

```

Comment:
Who the hell added pywin32 dependency to mac... It was probably me. Obviously copied from win and forgot to delete. Sorry.